source/releasenotes/about.rst

```diff
@@ -17,7 +17,8 @@
 What's New in |release|
 =======================
 
-Apache CloudStack |release| is a |version| LTS release with over 15 major new features, and over 200 enhancements and fixes since 4.13.  Highlights include:
+Apache CloudStack |release| is a |version| LTS release with over 15 major new features, and
+over 200 enhancements and fixes since 4.13.  Highlights include:
```

Comment:
Any line break/making those shorter in source docs will NOT affect the output - it's identical, but it doesn't hurt to make it beautiful.

Reply:
I restructured the entire documentation set; I'm fully aware that a new line at c.80 characters does not make any difference to the final  output.  However, we have a 'standard', so realigned the text while adding some explanation.